src/tests/Microsoft.VisualStudio.Composition.Tests/MetadataViewImplementationTests.cs

```diff
@@ -20,15 +20,17 @@ public void MetadataViewImplementationDirectQuery(IContainer container)
             Assert.IsType<MetadataViewClass>(export.Metadata);
             Assert.Equal("1", export.Metadata.A);
             Assert.Null(export.Metadata.B);
+            Assert.Equal(10, export.Metadata.PropertyWithMetadataThatDoesNotMatchType);
         }
 
-        [MefFact(CompositionEngines.V1)]
+        [MefFact(CompositionEngines.V1Compat)]
```

Comment:
I didn't understand why this wasn't tagged as V1Compat.  I was assuming this would need to be tested with V1 of MEF as well as VS MEF so I switched it.